src/accel/HepMC3PrimaryGenerator.hh

```diff
@@ -51,6 +51,7 @@ class HepMC3PrimaryGenerator final : public G4VPrimaryGenerator
     explicit HepMC3PrimaryGenerator(std::string const& filename);
 
     CELER_DELETE_COPY_MOVE(HepMC3PrimaryGenerator);
+    ~HepMC3PrimaryGenerator() override = default;
```

Comment:
Should these be final instead of override? (The class is final and the rest of the functions are too.) Or does it matter?

Reply:
When updating these, I tried to use final for final classes, and override instead. I missed that one